src/qt/menu.cpp

```diff
@@ -30,7 +30,7 @@ class wxQtActionGroup : public QActionGroup, public wxQtSignalHandler
 
 public:
     explicit wxQtActionGroup( wxMenu* handler )
-        : QActionGroup( handler->GetHandle() ),
+        : QActionGroup( handler != nullptr ? handler->GetHandle() : nullptr ),
```

Comment:
Same question as for the toolbar: is it really normal for `handler` to be null here?